src/less/wizard-stepper/base/wizard-stepper.less

```diff
@@ -0,0 +1,129 @@
+@mobile: ~"only screen and (max-width: 600px)";
+@wizard-step-confirmation-color: #36cf57;
+@wizard-step-default-color: #767676;
+
+.wizard-stepper-list {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    overflow-y: auto; // For non mobile view port widths. For mobile widths, we should fallback to vertical
+}
+
+.wizard-stepper-list__item {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    text-decoration: none;
+}
+
+.wizard-stepper-list__item--confirmation,
+.wizard-stepper-list__item--upcoming,
+.wizard-stepper-list__item--inprogress {
+    &:extend(.wizard-stepper-list__item);
+
+    margin-right: 55px;
+}
+
+.wizard-stepper-list__item--confirmation,
+.wizard-stepper-list__item--upcoming {
+    color: @wizard-step-default-color;
+}
+
+.wizard-stepper-list__item__node {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 30px;
+
+    &::before {
+        background-color: @wizard-step-default-color;
+        bottom: 35px;
+        content: '';
+        height: 3px;
+        left: -50px;
+        position: absolute;
+        width: 45px;
+    }
+}
+
+.wizard-stepper-list__item--confirmation,
+.wizard-stepper-list__item--inprogress {
+    .wizard-stepper-list__item__node::before {
+        background-color: @wizard-step-confirmation-color;
+    }
+}
+
+.wizard-stepper-list__item__title {
```

Comment:
BEM only goes as far as children, not grandchildren. So this would have to be `.wizard-stepper-list__item-title`. I know, it kind of feels like cheating, since title belongs to the item! But this is the way we have to do it.